app/interactors/send_timer_reminders.rb

```diff
@@ -0,0 +1,18 @@
+class SendTimerReminders
+  include Interactor
+  include HasHarvest
+
+  before do
+    context.date ||= Date.current
+  end
+
+  def call
+    User.all.each do |user|
+      next if user.timer_reminder_sent_on.try(:today?)
```

Comment:
Ok @laserlemon you're going to hate this and @trestrantham is going to love it :package: 

```ruby
User.where("timer_reminder_sent_on < ? OR timer_reminder_sent_on IS NULL", context.date).
  joins(
    User.arel_table.join(Day.arel_table, Arel::Nodes::OuterJoin).on(
      Day.arel_table[:user_id].eq(User.arel_table[:id]).and(
        Day.arel_table[:date].eq(context.date).and(Day.arel_table[:pto].eq(false))
      )
    ).join_sources
  )
```